android/src/main/java/com/reactcommunity/rndatetimepicker/Common.java

```diff
@@ -30,4 +42,32 @@ public static void dismissDialog(FragmentActivity activity, String fragmentTag,
       promise.reject(e);
     }
   }
+
+	public static int getDefaultDialogButtonTextColor(@NonNull Context activity) {
+		TypedValue typedValue = new TypedValue();
+		Resources.Theme theme = activity.getTheme();
+		theme.resolveAttribute(android.R.attr.textColorPrimary, typedValue, true);
+		@ColorRes int colorRes = (typedValue.resourceId != 0) ? typedValue.resourceId : typedValue.data;
+		@ColorInt int colorId = ContextCompat.getColor(activity, colorRes);
+		return colorId;
+	}
+
+	@NonNull
+	public static DialogInterface.OnShowListener ensureButtonsVisible(@NonNull Context activityContext, final AlertDialog dialog) {
```

Comment:
this will serve to allow customizing button text color in future

Reply:
nit, What about naming this method something like `styleButtons` or `updateButtonsStyle`?